src/_pytest/main.py

```diff
@@ -370,9 +370,18 @@ def pytest_runtestloop(session: Session) -> bool:
 def _in_venv(path: Path) -> bool:
     """Attempt to detect if ``path`` is the root of a Virtual Environment by
     checking for the existence of the pyvenv.cfg file.
-    [https://peps.python.org/pep-0405/]"""
+
+    [https://peps.python.org/pep-0405/]
+
+    for regression protection we also check for conda environments that do not include pyenv.cfg yet
+    https://github.com/conda/conda/issues/13337 is the conda issue tracking adding pyenv.cfg
```

Comment:
Let's link directly to the suggestion:

```suggestion
    For regression protection we also check for conda environments that do not include pyenv.cfg yet -- 
    https://github.com/conda/conda/issues/13337 is the conda issue tracking adding pyenv.cfg.
    
    Checking for the `conda-meta/history` file per https://github.com/pytest-dev/pytest/issues/12652#issuecomment-2246336902.
```

I was about to ask why `conda-meta/history` (before reading the thread), given it is possible to just remove the `history` file and conda still works -- I recall we explicitly deleting that file at work to workaround some bugs in `conda env`. 

However given it was a suggestion from @jezdez (a core conda maintainer) we should use it, of course.